UPGRADE-6.3.md

```diff
@@ -17,6 +17,8 @@ DependencyInjection
 DoctrineBridge
 --------------
 
+ * Deprecate `$subscriberIds` in `ContainerAwareEventManager` class constructor, use of doctrine listener is a preferred alternative way
+ * Deprecate `addEventSubscriber`, `removeEventSubscriber`and `initializeSubscribers` in `ContainerAwareEventManager` class, use of doctrine listener is a preferred alternative way
```

Comment:
I don't think we need to deprecate addEventSubscriber and addEventSubscriber
Also, initializeSubscribers is private so it doesn't need a deprecation notice

Reply:
I don't really get the point of keeping these two function if we are to deprecate passing doctrine subscribers to ContainerAwareEventManager. Could you please enlighten me about that?  🙏

Reply:
They're part of what the parent class allows to do - they're not specific to our implementation but part of a larger API.

Reply:
so I should completely remove the deprecation triggered in initializeSubscribers?

Reply:
I'm a bit confused as to whether I should also update the `subscribers` property, especially considering what is currently "allowed" and what the `initializeSubscribers` method handles, which is currently a mix  of both listeners and subscribers